src/topology/algebra/infinite_sum.lean

```diff
@@ -8,6 +8,7 @@ import algebra.big_operators.nat_antidiagonal
 import logic.encodable.lattice
 import topology.algebra.mul_action
 import topology.algebra.order.monotone_convergence
+import topology.algebra.matrix
```

Comment:
What do you think of making a new file for this? Then this file can be about abstract theory, and not particular data types.

Reply:
What should the file be called?

Reply:
I guess `*.matrix.infinite_sum` or `*.infinite_sum.matrix`. I don't have a strong preference.

Reply:
I went for `topology.instances.matrix` instead.